cli/flox/src/commands/containerize.rs

```diff
@@ -50,7 +68,15 @@ impl Containerize {
         };
 
         let (output, output_name): (Box<dyn Write + Send>, String) =
-            if output_path == Path::new("-") {
+            if let Some(registry) = self.load_into_registry {
+                let command = Command::new(&registry)
+                    .arg("load")
+                    .stdin(Stdio::piped())
+                    .spawn()
+                    .context("Could not start registry load command")?;
+
+                (Box::new(command.stdin.unwrap()), registry.to_string())
+            } else if output_path == Path::new("-") {
```

Comment:
nit: should we error if both `-f` and `--load-into-registry` are passed?

Reply:
We could, something as follows?

```console
$ flox containerize --load-into-registry=docker --file=bla.tar
ERROR: --load-into-registry and --file are mutually exclusive
```

Does bpaf have some mechanism to declare this, or do simple if statements at the top suffice?

Reply:
I can't remember if there's a way to do that with bpaf. I think if statements are fine or I can dig a little if that would help

Reply:
> I can't remember if there's a way to do that with bpaf.

Think in data structures. How in Rust you store either A or B but not both at once? You use an `enum`. Same here, make an `enum`, parse both sources in different branches (derive or combinatoric). If you pass both - `bpaf` will know that it can't represent both at once and will complain. https://github.com/pacak/bpaf/blob/e78ab789bce022744e5a49cc76db5b1f09b13295/examples/derive_this_or_that.rs

Reply:
@bryanhonof another example of that is here https://github.com/flox/flox/blob/db6407e5e7c3f84300844ed85e92bd1778a40672/cli/flox/src/commands/list.rs#L34-L55